icu4c/source/common/rbbi.cpp

```diff
@@ -848,7 +848,7 @@ int32_t RuleBasedBreakIterator::handleNext() {
 
        #ifdef RBBI_DEBUG
             if (gTrace) {
-                RBBIDebugPrintf("             %4ld   ", utext_getNativeIndex(&fText));
```

Comment:
What was the warning here? I think the %41d was just for the white-space padding.

Reply:
The warning here was "warning C4477: 'printf' : format string '%4ld' requires an argument of type 'long', but variadic argument 1 has type 'int64_t'", which is why I changed it to %lld.

Reply:
Oops, I thought that it was a "one" and didn't realize it was a "l". Thanks!